forms-flow-web/src/components/FOI/customComponents/Records/index.js

```diff
@@ -633,46 +855,63 @@ export const RecordsLog = ({
     setIsDownloadInProgress(false);
     setIsDownloadReady(false);
     setIsDownloadFailed(true);
-
-  }
+  };
 
   const downloadAllDocuments = async () => {
     let blobs = [];
     var completed = 0;
     let failed = 0;
-    var exporting = records.filter(record => !record.isduplicate)
+    var exporting = records.filter((record) => !record.isduplicate);
     for (let record of exporting) {
-      if (record.attachments) for (let attachment of record.attachments) {
-        if (!attachment.isduplicate) exporting.push(attachment);
-      }
+      if (record.attachments)
+        for (let attachment of record.attachments) {
+          if (!attachment.isduplicate) exporting.push(attachment);
+        }
     }
-    const toastID = toast.loading("Exporting files (" + completed + "/" + exporting.length + ")")
+    const toastID = toast.loading(
+      "Exporting files (" + completed + "/" + exporting.length + ")"
+    );
     try {
       for (let record of exporting) {
-        var filepath = record.s3uripath
-        var filename = record.filename
-        if (record.isredactionready && conversionFormats.includes(record.attributes?.extension?.toLowerCase())) {
+        var filepath = record.s3uripath;
+        var filename = record.filename;
+        if (record.isredactionready && record.isconverted) {
```

Comment:
removed conversionFormats.includes(record.attributes?.extension?.toLowerCase())) and replaced with record.isconverted